multi-currency/client/setup/tasks/add-currencies-task/index.js

```diff
@@ -27,9 +27,9 @@ import {
 
 // eslint-disable-next-line max-len
 // TODO: Migrate to 'multi-currency/settings/enabled-currencies-list/modal-checkbox-list'.
```

Comment:
Should we remove these TODOs?

Reply:
Good call. Fixed in [69be728](https://github.com/Automattic/woocommerce-payments/pull/9326/commits/69be728ae93069ec07212e27eef74bdc066b8c48).